src/main.jl

```diff
@@ -54,28 +53,45 @@ end
 ###############################################################################################################
 
 """
-		runopt(t1::Tree, t2::Tree; kwargs...)
+		runopt(t1::Tree, t2::Tree[, constraint]; kwargs...)
 		runopt(oa::OptArgs, t1::Tree, t2::Tree)
-		runopt(oa::OptArgs, trees::Dict{<:Any,<:Tree})
 
-Run optimization at constant γ. See `?Optargs` for arguments. In the first form, keyword
-  arguments are given to `OptArgs`.
+Run optimization at constant γ. See `?Optargs` for arguments.
+In the first form, keyword arguments are given to `OptArgs`.
+
+A `constraint` can be given in the form of an MCC where nodes that should be together are in
+ the same cluster.
+It will be used as `shared_branch` constraint while performing simulated annealing,
+ lowering the likelihood that nodes that should be in the same MCC are
+ split from each other.
 """
-runopt(t1::Tree, t2::Tree; kwargs...) = runopt(OptArgs(;kwargs...), t1, t2)
+function runopt(t1::Tree, t2::Tree, constraint=nothing; kwargs...)
```

Comment:
What could be really useful for me is an example of how this is called with a constraint, maybe in the docstring.